pkgs/tools/package-management/lix/default.nix

```diff
@@ -85,6 +85,8 @@ let
         nix-eval-jobs = self.callPackage (callPackage ./common-nix-eval-jobs.nix nix-eval-jobs-args) {
           stdenv = lixStdenv;
         };
+
+        nix-fast-build = self.callPackage ../../../../pkgs/by-name/ni/nix-fast-build/package.nix { };
```

Comment:
Is it possible to override with a new scope rather than do the `../../` dance?

Reply:
I think so; `self.nix-fast-build.override { nix = self.lix; }` or something like that.

In the future I think we want to have overlays which set `nix = lix` but I think this is fine for now.

Reply:
I went with the path import (as ugly as it looks) so that we get the automatic scope inheritance; yes we can manually `.override { nix = self.lix; }` but then we'd also have to remember to include `inherit (self) nix-eval-jobs;` and every other possible input which recursively might depend on something in the scope... _and_ if the original derivation changes we'd also have to keep up with it there...

But its also not fool proof so I'm open to switching with `.override` for now until there's a more general-purpose overlay available

Reply:
Updated, let me know if you had something different in mind!

Reply:
is it possible to override `nix=lix` on the whole of `packages` and inherit from that here as some sort of whitelist? e.g. `inherit (pkgsWithLix) nix-eval-jobs nix-fast-build;`

Reply:
That is however an increase in scope for the PR, i'm fine merging this until such a solution is worked out if ever

Reply:
@pbsds Yes, that's what the Lix NixOS module does, we want to get it upstreamed (and adding `lixPackageSets` has been in service of that goal) but it hasn't happened yet